pom.xml

```diff
@@ -29,15 +29,15 @@
 		<project.jdk.version>20</project.jdk.version>
 
 		<!-- Group IDs of jars that need to stay on the class path for now -->
-		<!-- Once hypfvieh, swiesend, purejava and integrations-linux have module-info, remove them-->
-		<nonModularGroupIds>org.ow2.asm,org.apache.jackrabbit,org.apache.httpcomponents,de.swiesend,org.purejava,com.github.hypfvieh</nonModularGroupIds>
+		<!-- Once swiesend has module-info, remove them-->
+		<nonModularGroupIds>org.ow2.asm,org.apache.jackrabbit,org.apache.httpcomponents</nonModularGroupIds>
```

Comment:
Yes, we're in a bit of a messy situation here: secret-service should not be put on the module path. Yet it does work, but is a very brittle setup. It only works, because it is the _only_ lib that uses package names outside of its own namespace. If there ever is a second dependency (regardless of whether direct or transitive or whether on `-p` or `-cp`) that conflicts with those packages, the JVM will refuse to work.

I would therefore prefer to keep it on the classpath, but apparently [there were problems with it](https://github.com/cryptomator/cryptomator/pull/2885#issuecomment-1562769338). Maybe worth checking again.

Reply:
I tested this again, just for sure. If `de.swiesend` stays on the classpath, starting the build apppimage termiantes instantly with  `module.FindException: Module de.swiesend not found, required by org.cryptomator.integrations-linux`.

This is expected behaviour, since integrations-linux is now also modular, and secret-service has to be on the module path, even if it is an automatic module. So yes, this is currently the only solution. (see also https://stackoverflow.com/a/46289257)

Reply:
I'll create a ticket in integrations-linux for the split package risk

Reply:
See https://github.com/cryptomator/integrations-linux/issues/26